README.md

```diff
@@ -172,27 +188,73 @@ command.
 
-#### Conversation ####
+Conversation
+
+- Comments on the pull request show up here, including any inline comments that
+  are added when viewing the file diffs. A summary of submitted commits is also
+  listed.
+- Use this section to submit general feedback about the pull request.
+
+Commits
 
-Comments on the pull request show up here, including any inline comments that
-are added when viewing the file diffs. A summary of submitted commits is also
-listed.
+- A more detailed list of commits included in the pull request, that will allow
+  a reviewer to view each individually.
+- Usually before merging, you'll want all the commits from that change to be
+  squashed into a single commit.
 
-Use this section to submit general feedback about the pull request.
+Files changed
 
-#### Commits ####
+- The `diff` view of the pull request. Lets the reviewer see the full patch in
+  its entirety, and add inline comments on changes.
+- Use this section to submit line-specific feedback.
 
-A more detailed list of commits included in the pull request, that will allow a
-reviewer to view each individually.
+#### Giving feedback ####
 
-#### Files changed ####
+Everyone is invited to give feedback regardless of experience. Comments can
+touch on:
 
-The `diff` view of the pull request. Lets the reviewer see the full patch in
-its entirety, and add inline comments on changes.
+- phrasing of any text added
+- tags on event posts (do they make sense? are they existing tags?)
+- lines that are over 80 characters long (which we avoid)
+- any identation that's hard to read 
+- the way the change looks on the [Heroku Review App](#heroku-review-apps)
+- if there are mutliple commmits that can be [squashed into one
+  commit](#squashing-commits)
 
-Use this section to submit line-specific feedback.
+You can even look at it and decide it looks good and just comment with “looks
+good to me!” 
+
+
+### Squashing commits ###
+
+To squash multiple commits created over the review process into a single commit
+that contains your change, use [git
+rebase](https://help.github.com/articles/about-git-rebase/).
+
+If you have `x` commits you want to squash into one, run `git rebase -i
+HEAD~x`. It will list the last `x` commits. Keep the first one as `pick` ___,
+and then choose `fixup` (or you can just type `f`) for the rest of the commits.
+Save and close, and there should just be one commit there! (To confirm, you can
+run `git log`.)
+
+When you push the new squashed commit, `git` will be confused and think you're
+missing those `x` commits (when really you just turned them all into a new
+single commit). It will suggest you run `git pull`. **Do not ever run `git
+pull`**. Instead run `git push --force`.
+
+### Pulling in changes ###
+
+Maybe while you were working on your change, someone else contributed updates
+to the repository and you want add them to your branch. **Do not ever run `git
+pull`**. Instead run:
+
+```
+git fetch upstream
```

Comment:
User might have to run `git remote add upstream https://github.com/blah.git` if remote isn't added yet.